Roles/Crewmate/Monarch.cs

```diff
@@ -20,8 +20,8 @@ internal class Monarch : RoleBase
 
     private static OptionItem KnightCooldown;
     private static OptionItem KnightMax;
-    
-    private static int KnightLimit = new();
+
+    private static Dictionary<byte, int> KnightLimit = [];
```

Comment:
e.g for this instance it would be `private int KnightLimit;` and in add just do Knightlimit = KnightMax.GetInt(); (it's secretly `this.KnightLimit` btw, the ide just detects it automatically in a non-static method.)